src/pages/Pomodoro/lib/SettingsContext.tsx

```diff
@@ -1,12 +1,11 @@
 import {createContext, useState} from "react";
-// import style from "../Logic/Settings.module.css";
 
-export const SettingContext = createContext({});
+export const SettingContext = createContext();
 
 const SettingsContextProvider = (props) => {
 
     const [pomodoro, setPomodoro] = useState(0);
```

Comment:
this var name is a bit confusing, but not critical, just for info :)
could just be a counter, setCounter